pxtblocks/fields/field_ledmatrix.ts

```diff
@@ -44,6 +44,7 @@ export class FieldLedMatrix extends FieldMatrix implements FieldCustom {
     private currentDragState_: boolean;
 
     protected clearSelectionOnBlur = true;
+    protected forceFocusVisible = true;
```

Comment:
Safari seems to have a problem in being able to determine `focus-visible` correctly. Notably after zooming the workspace. We can ignore what the browser thinks for this field editor as it is either in mouse mode or keyboard mode. The mouse cannot open the widget div for this field editor, and the use of the mouse on the dropdown instantly closes it.